packages/driver/src/cypress/screenshot.coffee

```diff
@@ -94,6 +125,22 @@ validate = (props, cmd, log) ->
 
     values.clip = clip
 
+  if padding = props.padding
+    isNumberOrArrayOfNumbers = (
```

Comment:
@chrisbreiding I thought that extracting this into a function at the module level was overkill, so I just inlined the implementation here instead.

Reply:
I still think `not (isNumber or isShorthandPadding)` would be clearer, since then the latter variable name captures that the value should be [conforming to the shape of short-hand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Margin_and_Padding_Properties).